### ISSUE TITLE
Type error with generated codes for ListBox

### ISSUE THREAD

**Issue by knyipab** [ImpulseAdventure/GUIslice-Builder#45]
Thank you for the contributors for building this excellent project! The builder tool really makes the life of building GUI for embedded device much easier. 

Here I found a bug with the code generation. Any code generated by the builder for ListBox gives type error at `(char*)&m_acListboxBuf1` of the line below when compiled with VSCode/PlatformIO (I'm not sure about the behavior of Arduino IDE, but it's not compiling on VSCode/PlatformIO): 
```
pElemRef = gslc_ElemXListboxCreate(&m_gui,E_ELEM_LISTBOX1,E_PG_MAIN,&m_sListbox1,
    (gslc_tsRect){30+2,190+4,200-23,100-7},E_FONT_TXT5,
    (char*)&m_acListboxBuf1,sizeof(m_acListboxBuf1),0);
```

By looking at the declaration of `gslc_ElemXListboxCreate`, `uint8_t*` should be passed to the function but not `char*`. I guess it has something to do with line 415 & 469 of GUIslice-Builder/builder/src/main/java/resources/templates/arduino.t and line 305 & 403 of GUIslice-Builder/builder/src/main/java/resources/templates/linux.t to fix the bug. 

It would be nice to see the bug fix ; ). Thanks!

**Comment by knyipab** [comment]
I think the bug could be fixed by just changing `(char*)` to `(uint8_t*)` for those few lines.

**Comment by Pconti31** [comment]
Assigned bug 122.  I made the changes and arduino ide worked ok so they will be in build 0.13.b010
Paul--

**Comment by Pconti31** [comment]
fixed in release 0.13.b010
